packages/osd-ui-framework/dist/kui_dark.css

```diff
@@ -588,33 +28,25 @@ main {
   margin-right: 25px; }
   .kuiBarSection:not(:first-child):not(:last-child):not(:only-child) {
     justify-content: center;
-    /* 3 */ }
+     }
   .kuiBarSection:first-child {
     margin-left: 0; }
   .kuiBarSection:last-child {
     margin-right: 0;
     flex: 0 1 auto;
-    /* 4 */
-    justify-content: flex-end;
-    /* 5 */ }
+        justify-content: flex-end;
+     }
```

Comment:
Something seems weird with the indentation in this diff - we shouldn't see changes, right? Does the strip tool make some assumptions about spacing?

Reply:
Maybe a future thing would be to check if the line is empty after removing the comment and delete the line if so

Reply:
I think `comment-stripper` should only do what it is asked to do and formatting of the document should be left the libraries whose job is that.

Reply:
Yeah, I don't think it matters, but I agree with Matt's suggestion that a line that contains only a comment should just be fully removed would be an improved behavior.